preview-panel/src/style.css

```
@@ -148,86 +148,112 @@ body {
 
 .ProseMirror pre code {
-  background: none;
-  padding: 0;
-  font-size: 0.9em;
-  font-family: "Cascadia Code", "Fira Code", "Consolas", monospace;
-  /* Base text colour (overridden per-token below) */
-  color: var(--hljs-base);
+    background: none;
+    padding: 0;
+    font-size: 0.9em;
+    font-family: "Cascadia Code", "Fira Code", "Consolas", monospace;
```

Stylelint (17.10.0)

[error] Expected no quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)